float32 motor_L_1
float32 motor_L_2
float32 motor_L_3
float32 motor_R_1
float32 motor_R_2
float32 motor_R_3
